lib/proxy/rpmsg_retarget.c

```diff
@@ -133,7 +133,6 @@ int rpmsg_rpc_send(struct rpmsg_rpc_data *rpc,
 		if (rpc->poll)
 			rpc->poll(rpc->poll_arg);
 	}
-	return ret;
```

Comment:
will you get compilation warning if you remove the return? it looks like you will need to have return as the return type is int.